docs/core/compatibility/containers/8.0/aspnet-port.md

```diff
@@ -7,27 +7,62 @@ ms.date: 07/12/2023
 
 The default ASP.NET Core port configured in .NET container images has been updated from port 80 to 8080.
 
-We also added the new `ASPNETCORE_HTTP_PORTS` environment variable instead of `ASPNETCORE_URLS`.
+We also added the new `ASPNETCORE_HTTP_PORTS` environment variable as a simpler alternative to `ASPNETCORE_URLS`.
```

Comment:
could you say why it's simpler? just  because it's a number instead or does it have different effects?

Reply:
also I'd move this line down into new behavior section perhaps? the first line is the headline.

Reply:
```suggestion
We also added the new `ASPNETCORE_HTTP_PORTS` environment variable as a simpler alternative to `ASPNETCORE_URLS`. The new variable expects a semicolon delimited list of ports numbers, while the older variable expects a more complicated syntax.
```

Reply:
looks good